packages/@tinacms/graphql/src/resolve.ts

```diff
@@ -162,11 +162,23 @@ export const resolve = async ({
             return value
           }
           if (info.fieldName === 'getCollections') {
+            const getCollectionNode = info.fieldNodes.find(
+              (x) => x.name.value === 'getCollections'
+            )
+            const hasDocuments = getCollectionNode.selectionSet.selections.find(
+              (x) => {
+                // @ts-ignore
+                return x?.name?.value === 'documents'
+              }
+            )
             return tinaSchema.getCollections().map((collection) => {
-              return resolver.resolveCollection(collection.name)
+              return resolver.resolveCollection(
+                collection.name,
+                Boolean(hasDocuments)
+              )
             })
           }
-          return resolver.resolveCollection(args.collection)
+          return resolver.resolveCollection(args.collection, true)
```

Comment:
Wouldn't we want to do the same here?

Reply:
@jeffsee55 I was not sure. Left it as true so that I would not break anything. What query would cause this code to run?

Reply:
Just updated to account for `getCollection` is there any other collection field?